vendor/assets/javascripts/behaviors/selectable_asset.js

```diff
@@ -0,0 +1,17 @@
+Cmsify.SelectableAsset = function(el) {
+  this.$el = $(el);
+  this.select = this.select.bind(this);
```

Comment:
Yeah, that can be a good option. I lean towards defining binding in a single location so that you can be certain that every call to the function will behave exactly the same way. And so you can see at a glance all the bound functions. But mostly putting them in a single place at the top of a class is a `React` design pattern that I'm just used to now. :)